client/src/components/Suggestions.tsx

```diff
@@ -0,0 +1,32 @@
+import "./suggestions.styles.css";
```

Comment:
This looks good to me! My only suggestion is renaming this to suggestions.css since we know based on the .css that these are styles.

Reply:
Agreed - feel free to merge this in once that change is made :)